src/UI/citizens/components/select-date-time/CitizensSelectDateTime.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+
+import {
+  Container,
+  Krug,
+  Broj,
+  Datum,
+  Oznacite,
+  Podebljano,
+  OdaberiteDatum,
+  OdaberiteVrijeme,
+  Pocetak,
+  Kraj,
+  Unos,
+  Rectangle,
+  Vector,
+  Input,
+  Timeline,
+  Time,
+  Prazno,
+  TimeDivide,
+  Zauzeto,
+  Slobodno,
+  Zauzeto1,
+  Legenda1,
+  Legenda2,
+  LegendaBoja,
+  LegendaText
+} from './StyleCitizensSelectDateTime';
+
+var poc = [
```

Comment:
Provjeriti sa Matijom u kojem formatu API šalje date-time info za pojedini dan. Nakon toga stvoriti dummy objekt koji odgovara 1-1 tom JSON-u koji ćemo primati sa API-ja. Mislim da početak i kraj neće biti ovako razdvojeni, definitivno će to biti property unutar istog objekta... Ako odmah implementiraš ovako, kasnije lako prištekamo data flow iz API-ja umjesto ovog dummy-ja.